pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole_controller.go

```diff
@@ -52,26 +52,26 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// Watch for changes to secondary resource DeploymentConfig and requeue the owner MobileDeveloperConsole
```

Comment:
Any particular reason for the reordering here?

Reply:
The watch for `Service` was missing, so I just sort of re-ordered them all based on their groups while I was adding that: the Kubernetes core ones are first, then the other k8s groups, then the OpenShift ones are at the bottom.

Reply:
ok, sounds great. feel free to merge